app/mailers/flood_risk_engine/enrollment_mailer.rb

```diff
@@ -0,0 +1,19 @@
+module FloodRiskEngine
+  class EnrollmentMailer < ActionMailer::Base
+    add_template_helper(EmailHelper)
+    layout "flood_risk_engine/layouts/mail"
+
+    def submitted(enrollment_id:, recipient_address:)
+      i18n_scope = "flood_risk_engine.enrollment_mailer.submitted"
+      subject = I18n.t(".subject", scope: i18n_scope)
+      underlying_enrollment = FloodRiskEngine::Enrollment.find(enrollment_id)
```

Comment:
`Enrollment.find(enrollment_id)` should work as EnrollmentMailer is within the FloodRiskEngine namespace.